app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api.v1::GreetingsController < ApplicationController
+  # GET /greetings
+  # GET /greetings.json
+  def index
+    @greetings = Greeting.all.order(Random.rand(Greeting.count)).first
+    render json: @greetings
+  end
```

Comment:
- [ ] Your app does not return a random message. It returns the same message for me all the time. Kindly fix that.

- [ ] If I refresh your app a number of times I get Internal Server errors. Kindly fix this.

  ![error](https://user-images.githubusercontent.com/81468675/183676567-d92769d2-738c-4ea8-a184-7e70481e90f9.jpg)

- [ ] **[OPTIONAL]** I would recommend you return only the `message` value, and not the entire database attribute.

  ![msg-1](https://user-images.githubusercontent.com/81468675/183676839-340e7618-e8cd-48df-bf99-7d0a9922489c.jpg)

  ### HINT:
  ```
  def index
    @greetings = Greeting.all.sample.message
    render json: @greetings
  end
  ```